components/Common/Pagination/useGetPageElements.tsx

```diff
@@ -0,0 +1,146 @@
+'use client';
+import Link from 'next/link';
```

Comment:
Could you use our `LocalizedLink` instead of `next/link`?

Reply:
Yes, sure! I will replace them